osu.Game/Overlays/Toolbar/ToolbarMusicButton.cs

```diff
@@ -45,14 +45,15 @@ private void load(NowPlayingOverlay music)
                 Height = IconContainer.Height,
                 Margin = new MarginPadding { Horizontal = 2.5f },
                 Masking = true,
-                Children = new[]
+                CornerRadius = 3f,
```

Comment:
This looks incorrect when not expanded. Should make `volumeDisplay` a `CircularContainer` for automatic corner radius.